tests/integration/hive_metastore/test_migrate.py

```diff
@@ -101,3 +102,54 @@ def test_migrate_external_table(ws, sql_backend, inventory_schema, make_catalog,
 
     target_tables = list(sql_backend.fetch(f"SHOW TABLES IN {dst_schema.full_name}"))
     assert len(target_tables) == 1
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=5))
+def test_revert_migrated_table(ws, sql_backend, inventory_schema, make_schema, make_table, make_catalog):
+    src_schema1 = make_schema(catalog_name="hive_metastore")
+    src_schema2 = make_schema(catalog_name="hive_metastore")
+    table_to_revert = make_table(schema_name=src_schema1.name)
+    table_not_migrated = make_table(schema_name=src_schema1.name)
+    table_to_not_revert = make_table(schema_name=src_schema2.name)
+    all_tables = [table_to_revert, table_not_migrated, table_to_not_revert]
+
+    dst_catalog = make_catalog()
+    dst_schema1 = make_schema(catalog_name=dst_catalog.name, name=src_schema1.name)
+    dst_schema2 = make_schema(catalog_name=dst_catalog.name, name=src_schema2.name)
+
+    static_crawler = StaticTablesCrawler(sql_backend, inventory_schema, all_tables)
+    tm1 = TablesMigrate(static_crawler, ws, sql_backend, dst_catalog.name)
+    tm1.migrate_tables()
+
+    tables_after_migration = []
+    for migrated_table in tm1._seen_tables.keys():
+        src_table_elements = tm1._seen_tables[migrated_table].split(".")
+        table = TableInfo(
+            catalog_name=src_table_elements[0],
+            schema_name=src_table_elements[1],
+            name=src_table_elements[2],
+            table_type=TableType.MANAGED,
+            data_source_format=DataSourceFormat.DELTA,
+            properties={"upgraded_to": migrated_table},
+        )
+        tables_after_migration.append(table)
```

Comment:
Invoke the real migration methods from TablesMigrate class (not this fake), otherwise we'll break the revert logic without noticing when we improve that functionality.

Those methods exist already.

Reply:
tell me where do you need assessment to call this method? https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/hive_metastore/tables.py#L222-L244

Reply:
https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/hive_metastore/tables.py#L207-L220 this one, to be precise.

Reply:
Fixed